CeCILL-C.txt

```diff
@@ -0,0 +1,14 @@
+Copyright (C) 2006-2015 INRIA and contributors
```

Comment:
CeCILL-C.txt contains the template of the license. LICENSE.txt contains the full text of the license. This isn't the same thing but I can rename CeCILL-C.txt to template_license.txt if you want.